decidim-admin/app/controllers/decidim/admin/newsletters_controller.rb

```diff
@@ -7,10 +7,11 @@ class NewslettersController < Decidim::Admin::ApplicationController
       include Decidim::NewslettersHelper
       include Decidim::Admin::NewslettersHelper
       include Paginable
-      helper_method :newsletter
+      helper_method :newsletter, :newsletter_attention_callout_args, :newsletter_recipients_count_callout_args
```

Comment:
why are `:newsletter_attention_callout_args` and `:newsletter_recipients_count_callout_args` helper methods here?